### ISSUE TITLE
Error on Packaging

### ISSUE THREAD

**Issue by Angelo-Gebauer** [microsoft/MixedReality-UXTools-Unreal#16]
## UX Tools for Unreal v0.10.0

> **UATHelper: Packaging (HoloLens):   LogPython: Error: '/Script/UXTools.EUxtPaginateResult' has an invalid 'ScriptName' meta-data value 'Failed, Concurrent Operation': Name contains ',' which is invalid for Python.**

> **UATHelper: Packaging (HoloLens):   LogPython: Error: '/Script/UXTools.EUxtPaginateResult' has an invalid 'ScriptName' meta-data value 'Failed, Concurrent User Interaction': Name contains ',' which is invalid for Python.**

> **UATHelper: Packaging (HoloLens):   LogPython: Error: '/Script/UXTools.EUxtScrollDirection' has an invalid 'ScriptName' meta-data value 'Up And Down': Name contains ' ' which is invalid for Python.**

> **UATHelper: Packaging (HoloLens):   LogPython: Error: '/Script/UXTools.EUxtScrollDirection' has an invalid 'ScriptName' meta-data value 'Left And Right': Name contains ' ' which is invalid for Python.**

> **PackagingResults: Error: '/Script/UXTools.EUxtPaginateResult' has an invalid 'ScriptName' meta-data value 'Failed, Concurrent Operation': Name contains ',' which is invalid for Python.**

> **PackagingResults: Error: '/Script/UXTools.EUxtPaginateResult' has an invalid 'ScriptName' meta-data value 'Failed, Concurrent User Interaction': Name contains ',' which is invalid for Python.**

> **PackagingResults: Error: '/Script/UXTools.EUxtScrollDirection' has an invalid 'ScriptName' meta-data value 'Up And Down': Name contains ' ' which is invalid for Python.**

> **PackagingResults: Error: '/Script/UXTools.EUxtScrollDirection' has an invalid 'ScriptName' meta-data value 'Left And Right': Name contains ' ' which is invalid for Python.**

**Comment by luis-valverde-ms** [comment]
Thanks for the info @Angelo-Gebauer . I take it that this happens to you when trying to package for HoloLens a project that uses the UXT plugin? What version of UE are you using? Can you send us the full log?

**Comment by Angelo-Gebauer** [comment]
Hi, yes we try to package for HoloLens. We're using 4.25.3 We didnt have this problem with the old version of the plugin.
Log:
[Log.txt](https://github.com/microsoft/MixedReality-UXTools-Unreal/files/5536179/Log.txt)

**Comment by luis-valverde-ms** [comment]
Can you try replacing 'ScriptName' by 'DisplayName' in all the enum values in `EUxtPaginateResult` and `EUxtScrollDirection`? You should end up with something like this:

```
UENUM(BlueprintType)
enum class EUxtScrollDirection : uint8
{
	UpAndDown		UMETA(DisplayName = "Up And Down", Tooltip = "The menu will scroll vertically. In this case Tiers will specify the number of rows, and Viewable area will specify the number of columns that are viewable. The collection will populate by filling columns in order."),
	LeftAndRight		UMETA(DisplayName = "Left And Right", Tooltip = "The menu will scroll horizontally. In this case Tiers will specify the number of columns, and Viewable area will specify the number of rows that are viewable. The collection will populate by filling rows in order."),
};

UENUM(BlueprintType)
enum class EUxtPaginateResult : uint8
{
	Success				UMETA(DisplayName = "Success", Tooltip = "The operation completed successfuly."),
	Failed_ConcurrentOperation	UMETA(DisplayName = "Failed, Concurrent Operation", Tooltip = "The operation was aborted due to another move request that has not yet finished."),
	Failed_ConcurrentInteraction	UMETA(DisplayName = "Failed, Concurrent User Interaction", Tooltip = "The operation was aborted because there is another interaction with the object." ),
};
```

**Comment by Angelo-Gebauer** [comment]
Sorry, for now we will just use the old version. We don't really have time to test things.

**Comment by luis-valverde-ms** [comment]
Resolved by Vanessa's fix: 2afc85b
